### DISCUSSION TITLE
Feature Request: Backlight Level Save Points

### DISCUSSION THREAD

**Discussion by katzenbeisser** [koreader/koreader#8960] [Ideas]
Since adjusting the levels of backlight with a slider can be pretty finicky on an e-ink screen, I'd like to suggest adding in save points for backlight levels.
That way you could save, for example, three different levels of backlight for when you're in different levels of surrounding light. Then you could easily select an approriate backlight from the bottom or top menu, without having to mess around with the slider.

It would be especially helpful for reading in bed where it is completely dark, since I only need a light level of around 4 or so, and going to far puts the backlight at 0, which means I can't find the slider anymore to increase it.

**Comment by dosimple** [comment]
Might I suggest a workaround for a minimal use case? Gestures can be bound to set backlight level. Even multiple settings can be changed via single gesture.
